src/rt/cover.c

```diff
@@ -1304,37 +1318,62 @@ static void cover_print_hierarchy_summary(FILE *f, cover_stats_t *stats, ident_t
 
    fprintf(f, "   </tr>\n");
 
+   float perc_stmt = 0;
+   float perc_branch = 0;
+   float perc_toggle = 0;
+   float perc_expr = 0;
+
+   if (stats->total_stmts > 0)
+      perc_stmt = 100.0 * ((float)stats->hit_stmts) / stats->total_stmts;
+   if (stats->total_branches > 0)
+      perc_branch = 100.0 * ((double)stats->hit_branches) / stats->total_branches;
```

Comment:
Why are some of the `double` and others `float`? It's better to just use `double` for everything.